eth/package.json

```diff
@@ -7,7 +7,7 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
-    "@textile/eth-storage": "^0.0.9",
+    "@textile/eth-storage": "^0.1.0",
```

Comment:
if you kept it 0.0.x it would signal that we may not be ready to maintain backward compatibility. just an idea though

Reply:
yeh, well, we've already released it :P but also, near-storage is already way up at 7 major, so I figured we might as well start with some semblance of semver now?